catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -139,6 +141,39 @@ public int hashCode() {
 		return userDefinedBrickList;
 	}
 
+	public UserDefinedBrick getUserDefinedBrick(UserDefinedBrick userDefinedBrick) {
```

Comment:
Maybe you could try to do this in a more generic way, because these three functions are nearly duplicate code